system/netdata.logrotate.in

```diff
@@ -0,0 +1,13 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		systemctl restart netdata.service
```

Comment:
You should never use `systemctl` or `/etc/init.d` directly but instead use the `service` wrapper:
`service netdata restart`